### DISCUSSION TITLE
When I use dp.Report().save(), the local html cannot be opened anymore.

### DISCUSSION THREAD

**Discussion by DaweiZhuCode** [datapane/datapane#379] [Help]
### Summary

When I use dp.Report().save(), the local html cannot be opened anymore.
What I see instead of the full report:
![image](https://github.com/datapane/datapane/assets/113312841/122c6ee0-dad7-48fb-8c6b-47b8b7cd9320)

I tried on the latest versions of Edge and Chrome.

Thanks!

**Comment by mands** [comment]
Hi @DaweiZhuCode - can you provide some more details on the problem or a code sample?
